Formula/jetty-runner.rb

```diff
@@ -17,6 +17,8 @@ def install
   end
 
   test do
-    assert_match version.to_s, shell_output("#{bin}/jetty-runner --version", 1)
```

Comment:
I've pushed the change. I've forgot about shell output redirect, thanx :). 
Regarding the test, it's tricky with jetty-runner, since it always requires Java .war file (package) to be run. I'll try to look for a better way of testing the formula, hopefully before next version of the jetty.